fabric-network/src/impl/event/transactioneventhandler.ts

```diff
@@ -169,7 +169,7 @@ export class TransactionEventHandler implements TxEventHandler {
 			.map((peer) => peer.name)
 			.join(', ');
 		const errorInfo = {
-			message: 'Event strategy not satisfied within the timeout period. No response received from peers: ' + unrespondedPeerNames,
+			message: `Event strategy not satisfied within the timeout period of ${util.inspect(this.options.commitTimeout)} ms. No response received from peers:  ${unrespondedPeerNames}`,
```

Comment:
Just `${String(this.options.commitTimeout)}` might be simpler. `util.inspect()` seems a bit overkill to convert something we know is a number into a string

Reply:
The `commitTimeout` value is in seconds (it is multiplied by 1000 to get the millisecond value for the Node timeout).